packages/db/mocks/challenges.mock.ts

```diff
@@ -18,6 +18,12 @@ export interface InfoFile {
 
 const redundantChallenges = ['pick', 'flatten'];
 
+const tsconfigOverrides: Record<string, object> = {
+  'assert-array-index': {
```

Comment:
isnt this pulled from the tsconfig ingest?

Reply:
oh wait does ts challenges not do that

Reply:
As mentioned in the PR description, technically there is a way to automatically ingest the tsconfigs for all type challenges by fetching the "play" URL for each and every one of them, then parsing them from the TypeScript playground URL they redirect you to, but I know for a fact that this is the only challenge that requires a custom tsconfig as of today.

I tried fetching every type challenge's playground URL, but it was awfully slow, so I don't think there's a point in doing this right now.

Reply:
that was my bad i just looked at the code and not the description 😂